templates/index.json

```diff
@@ -53,7 +53,7 @@
         "heading": {
           "type": "heading",
           "settings": {
-            "heading": "<p>Talk about your brand</p>",
+            "heading": "Talk about your brand",
```

Comment:
One question I have: Now that this setting uses `inline_richtext` instead of `richtext`, these need to be removed to pass tests. Is this going to cause an issue with existing stores that might have stored content that includes `<p>` tags? 🤔

Reply:
I am also wondering about the scenario merchants did add line breaks in that setting value `<br>` what will be their experience in how we map it out for a success theme update flow?

I mean my understanding it would make all <br> value invalid but would ignore them and do this:

>
    <p>This is a <br>
    sentence <br>
    written in <br>
    multiple lines</p>


to this :point_down:

>
    This is a sentence written in multiple lines


The only question I have is when I interact with the `inline_richtext` and try to add `<br>` manually and hit save, I end up in an error loop. I wonder if that’ll be the case for merchants.

Reply:
We are all good finally, we can ignore 👼 the above ☝️

Reply:
Ah ok, got it!